> The unauthenticated git protocol on port 9418 is no longer supported.

diff --git a/test/test_remote.py b/test/test_remote.py
index 7b9d192..89280df 100644
--- a/test/test_remote.py
+++ b/test/test_remote.py
@@ -165,6 +165,7 @@ def test_remote_list(testrepo):
     assert remote.name in [x.name for x in testrepo.remotes]
 
 @utils.requires_network
+@pytest.mark.skip(reason='github no longer supports the anonymous git protocol over port 9418')
 def test_ls_remotes(testrepo):
     assert 1 == len(testrepo.remotes)
     remote = testrepo.remotes[0]
